tardis/simulation/tests/test_simulation.py

```diff
@@ -88,10 +88,26 @@ def simulation_one_loop(
     ],
 )
 def test_plasma_estimates(simulation_one_loop, refdata, name):
-    try:
-        actual = getattr(simulation_one_loop.transport, name)
-    except AttributeError:
+    if name in ["nu_bar_estimator", "j_estimator"]:
+        actual = getattr(
+            simulation_one_loop.transport.transport_state.estimators, name
+        )
+    elif name in ["t_radiative", "dilution_factor"]:
         actual = getattr(simulation_one_loop.simulation_state, name)
+    elif name in ["output_nu", "output_energy"]:
+        OLD_TO_NEW_DICT = {
```

Comment:
Is this to avoid updating current refdata?

Reply:
yes